### ISSUE TITLE
Final report review

### ISSUE THREAD

**Issue by caolihe** [dchernysheva/ORIE-4741#12]
Likes:
1. very clear and reasonable explanation of feature selection
2. it is a good point that you noticed that it is not proper to directly apply cross-validation on this dataset, so you modify the cross-validation method with a rolling version in your project to adapt to the data.
3. the structure of the report is clear and has a rigorous logic in each part 
3. the potential improvements is well written and really show some thoughts about how to refine the project.

Improvements:
1. For different industry, the factors affect the stock market in COVID time might be quite different, maybe more analysis for this in feature selection section should be included. 
2. It will better if including some explanations about why the models are chosen. For example, why quadratic loss and regularization are used for some companies and huber loss and one regularizer are used for some other companies. Maybe cross-validation can also be applied there to see which loss function and regularizer should be chosen. 
3. when doing cross-validation to choose hyper-parameters, it would be better if it is done for 2 rounds. In the first round, the parameters can be tested in a large range, for example, lambda from 0.01 to 100 with a large step size. For the second round, it can be done in a smaller range around the best hyper parameters get from the first round.
